spec/lib/js_locale_helper_spec.rb

```diff
@@ -262,7 +262,9 @@ def clear_cache!
   end
 
   describe ".output_client_overrides" do
-    it "should not output deprecated translation overrides" do
```

Comment:
In RSpec, it’s better to assert that something is true instead of saying that it _should_ be true (otherwise anyway, that means that’s a bug :sweat_smile:)